nuxeo-tree-snapshot-core/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.nuxeo.snapshot</groupId>
+    <artifactId>nuxeo-tree-snapshot</artifactId>
+    <version>8.10.1-SNAPSHOT</version>
```

Comment:
nuxeo-tree-snapshot parent in`8.10-SNAPSHOT` is not using this `8.10.1-SNAPSHOT` version